packages/web-forms/src/components/common/map/useMapBlock.ts

```diff
@@ -0,0 +1,282 @@
+import {
+	getSavedStyles,
+	getSelectedStyles,
+	getUnselectedStyles,
+} from '@/components/common/map/map-styles.ts';
+import type { FeatureCollection } from 'geojson';
+import { Map, MapBrowserEvent, View } from 'ol';
+import { Zoom } from 'ol/control';
+import Feature from 'ol/Feature';
+import GeoJSON from 'ol/format/GeoJSON';
+import { LineString, Point, Polygon } from 'ol/geom';
+import TileLayer from 'ol/layer/Tile';
+import WebGLVectorLayer from 'ol/layer/WebGLVector';
+import type { Pixel } from 'ol/pixel';
+import { fromLonLat } from 'ol/proj';
+import { OSM } from 'ol/source';
+import VectorSource from 'ol/source/Vector';
+import { computed, ref, shallowRef, watch } from 'vue';
+
+type GeometryType = LineString | Point | Polygon;
+
+const STATES = {
+	LOADING: 'loading',
+	READY: 'ready',
+	ERROR: 'error',
+} as const;
+
+const DEFAULT_GEOJSON_PROJECTION = 'EPSG:4326';
+const DEFAULT_VIEW_CENTER = [0, 0];
+const MAX_ZOOM = 16;
+const MIN_ZOOM = 2;
+const GEOLOCATION_TIMEOUT_MS = 10 * 1000;
+const ANIMATION_TIME = 1000;
+const SMALL_DEVICE_WIDTH = 576;
+const MAP_HIT_TOLERANCE = 5;
```

Comment:
I always manage to select points and shapes when I click on them, but somehow I often miss lines. I click a little too far away, and the selection doesn't go through. Might just be something I'm doing wrong. 😅 But what do you think about increasing the hit tolerance even if just by a pixel or two?

I know you've been updating feature styles as well. Do you think those changes could affect this? I could wait to try it out with the new styles and see if it's still happening for me. Or we could see if the QA team mentions anything.

Reply:
This hit tolerance wasn't effective. The click only responds to "visible" elements. I've added a style for line features that expands the hit area by including an almost transparent stroke on top of the blue stroke. I've just pushed the change.